queue-workers/handler.js

```diff
@@ -0,0 +1,48 @@
+const appInsights = require('applicationinsights');
+const logger = require('../utils/logger');
+const resUtil = require('../utils/res-util');
+const createTelegramBot = require('../utils/createTelegramBot');
+appInsights.setup(process.env.APPINSIGHTS_CONNECTIONSTRING).start();
+
+
+module.exports = async function (message, context) {
+    const { name, data } = message;
+    const { body, bot_token } = data;
+
```

Comment:
We could merge both  into one for brevity
const { name, data: {body , botToken}} = message;
Remember we are renaming bot_token to botToken as per naming convention